src/file-utils/edfi_lms_file_utils/file_reader.py

```diff
@@ -107,14 +138,22 @@ def get_all_sections(
     """
     file = fr.get_sections_file(base_directory)
 
+    data_types = {
+        "SISSectionIdentifier": "string",
+        "Title": "string",
+        "SectionDescription": "string",
+        "Term": "string",
+        "Status": "string",
+    }
```

Comment:
An example of mapping data types, with no need to specify the universal data types.